dotnet/src/webdriver/CookieJar.cs

```diff
@@ -62,18 +96,21 @@ public void AddCookie(Cookie cookie)
         /// Delete the cookie by passing in the name of the cookie
         /// </summary>
         /// <param name="name">The name of the cookie that is in the browser</param>
-        public void DeleteCookieNamed(string name)
+        public void DeleteCookieNamed(string? name)
         {
-            Dictionary<string, object> parameters = new Dictionary<string, object>();
-            parameters.Add("name", name);
-            this.driver.InternalExecute(DriverCommand.DeleteCookie, parameters);
+            if (name is not null)
```

Comment:
This is safe because the current behavior of `DeleteCookieNamed(null)` does nothing. Easier to do this than to make parameter values nullable.

Reply:
Also lines up with below method which does a null check